packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts

```diff
@@ -201,9 +211,14 @@ export class FilamentViewer extends LitElement {
       this[$ibl] = ibl
       ibl.setIntensity(1.0);
       ibl.setRotation([0, 0, -1, 0, 1, 0, 1, 0, 0]);  // 90 degrees
+
       if (scenario.renderSkybox) {
         this[$skybox] = this[$engine].createSkyFromKtx(skyboxUrl);
         this[$scene].setSkybox(this[$skybox]);
+      } else {
+        this[$view].setBlendMode(View$BlendMode.TRANSLUCENT);
```

Comment:
Transparent routes works perfectly well for me. I did try use this[$skybox].setColor([0,0,0,1]), it didn't work and the background is still skybox's color. But if you don't think this is a problem then just ignore it.

Reply:
This seems fine for now.